docs/source/noise_models.rst

```diff
@@ -523,3 +573,270 @@ good starting point.**
 
 .. image:: images/GateNoiseModels_20_1.png
 
+Adding T1 and T2 type noise to all your gates
+---------------------------------------------
+
+Here we will provide some simple tools to define gate models for a full
+gate set (excluding :math:`RZ(\theta)` rotations, which can be executed
+without noise). We will again assume a very simple noise model in which
+the ideal gate is followed by the application of first dephasing and
+then damping noise.
+
+.. code:: ipython2
+
+    def combine_kraus_maps(k1, k2):
+        """
+        Generate the Kraus map corresponding to the composition
+        of two maps on the same qubits with k1 being applied to the state
+        after k2.
+
+        :param list k1: The list of Kraus operators that are applied second.
+        :param list k2: The list of Kraus operators that are applied first.
+        :return: A combinatorially generated list of composed Kraus operators.
+        """
+        return [np.dot(k1j, k2l) for k1j in k1 for k2l in k2]
+
+    def damping_after_dephasing(T1=30e-6, T2=30e-6, t_gate=50e-9):
+        """
+        Generate the Kraus map corresponding to the composition
+        of a dephasing channel followed by an amplitude damping channel.
+
+        :param float T1: The amplitude damping time
+        :param float T2: The dephasing time
+        :param flaot t_gate: The gate duration.
+        :return: A list of Kraus operators.
+        """
+        return combine_kraus_maps(damping_channel(t_gate/T1), dephasing_kraus_map(t_gate/T2))
```

Comment:
where is damping_channel defined?

Reply:
above but was missing in this rst version